array-methods-basic/basic-filter.js

```diff
@@ -4,8 +4,15 @@
 3. Write tests to cover the case where length is provided
  */
 
-var MESSAGE_LENGTH = 30;
-
+const MESSAGE_LENGTH = 30;
```

Comment:
the constant of message length should be inside the function since you're using it there! I wouldn't make it a global variable.
```javascript
const getShortMessages = (messages, length)=>{
    const MAX_LENGTH = length ? length : 10; //get the length, if not specified return 10
    const result = messages.filter((message)=>{
        return message.text.length < MAX_LENGTH;
    });
    return result;
};
```